run_labelling.sh

```diff
@@ -0,0 +1,5 @@
+
+python3 server.py \
+ --host 10.26.10.66 \
+ --port 5003 \
+ --dir /home/nlunscher/test_data/nolan/lidar_data_extracted/dataset_labelling_info
```

Comment:
I'm sure you didn't intent to commit this? :P

Reply:
nope lol my bad